src/qt/guiutil.cpp

```diff
@@ -879,4 +885,25 @@ int TextWidth(const QFontMetrics& fm, const QString& text)
 #endif
 }
 
+void LogQtInfo()
+{
+#ifdef QT_STATIC
+    const std::string qt_link{"static"};
+#else
+    const std::string qt_link{"dynamic"};
+#endif
+#ifdef QT_STATICPLUGIN
+    const std::string plugin_link{"static"};
+#else
+    const std::string plugin_link{"dynamic"};
+#endif
+    LogPrintf("Qt %s (%s), plugin=%s (%s)\n", qVersion(), qt_link, QGuiApplication::platformName().toStdString(), plugin_link);
+    LogPrintf("System: %s, %s\n", QSysInfo::prettyProductName().toStdString(), QSysInfo::buildAbi().toStdString());
+    const QList<QScreen*> screens = QGuiApplication::screens();
+    for (int i = 0; i < screens.size(); ++i) {
```

Comment:
`for (const QScreen* s : QGuiApplication::screens())`?

Reply:
@promag 
Are these concerns relevant https://stackoverflow.com/questions/35811053/using-c11-range-based-for-loop-correctly-in-qt ?

Reply:
It's fine here, these are pointers.

Reply:
Implemented in the latest push.